deploy/chart/catalog_resources/rh-operators/federationv2.crd.yaml

```diff
@@ -0,0 +1,1396 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    api: ""
+    kubebuilder.k8s.io: 0.1.10
+  name: clusters.clusterregistry.k8s.io
+spec:
+  group: clusterregistry.k8s.io
+  names:
+    kind: Cluster
+    plural: clusters
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          type: string
+        kind:
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            authInfo:
+              properties:
+                controller:
+                  properties:
+                    kind:
+                      type: string
+                    name:
+                      type: string
+                    namespace:
+                      type: string
+                  type: object
+                user:
+                  properties:
+                    kind:
+                      type: string
+                    name:
+                      type: string
+                    namespace:
+                      type: string
+                  type: object
+              type: object
+            kubernetesApiEndpoints:
+              properties:
+                caBundle:
+                  items:
+                    type: byte
+                  type: string
+                serverEndpoints:
+                  items:
+                    properties:
+                      clientCIDR:
+                        type: string
+                      serverAddress:
+                        type: string
+                    type: object
+                  type: array
+              type: object
+          type: object
+        status:
+          properties:
+            conditions:
+              items:
+                properties:
+                  lastHeartbeatTime:
+                    format: date-time
+                    type: string
+                  lastTransitionTime:
+                    format: date-time
+                    type: string
+                  message:
+                    type: string
+                  reason:
+                    type: string
+                  status:
+                    type: string
+                  type:
+                    type: string
+                type: object
+              type: array
+          type: object
+      type: object
+  version: v1alpha1
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: null
+---
```

Comment:
Unfortunately these need to be split out into separate files at the moment

Reply:
Yikes! There are a lot of CRDs.